doc/manifest/schema/1.0.0/installer.md

```diff
@@ -0,0 +1,444 @@
+
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
+</details>
+
+### Installer
+<details>
+ <summary>Package installer</summary>
+
+ #### Required Field
+
+ The key represents an installer for a package.
+
+ >Note: Many of the keys related to installers may either be at the root level of the manifest, or included in an installer. Any values provided at the root level and not specified in an installer will be inherited.
+
+</details>
+
+### InstallerLocale
+<details>
+ <summary>Locale for package installer</summary>
+
+ #### Optional Field
+
+ This key represents the locale for an installer *not* the package meta-data. Some installers are compiled with locale or language specific properties. If this key is present, it is used to represent the package locale for an installer.
+ 
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
+
+</details>
+
+### Architecture
+<details>
+ <summary>The installer target architecture</summary>
+
+ #### Required Field
+
+ This key represents the hardware architecture targeted by the installer. The Windows Package Manager will attempt to determine the best architecture to use. If emulation is available and the native hardware architecture does not have a supported installer, the emulated architecture may be used.
+ </details>
+
+### InstallerUrl
+<details>
+ <summary>The installer Url</summary>
+
+ #### Required Field
+
+ This key represents the URL to download the installer.
+
+</details>
+
+### InstallerSha256
+<details>
+ <summary>Sha256 is required. Sha256 of the installer</summary>
+
+ #### Required Field
+
+ This key represents the SHA 256 hash for the installer. It is used to confirm the installer has not been modified. The Windows Package Manager will compare the hash in the manifest with the calculated hash of the installer after it has been downloaded.
+
+ >Note:  The [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) can be used to determine the SHA 256 of the installer. The `winget hash &lt;pathToInstaller&gt;` command can also be used to determine the SHA 256 of the installer.
+
+</details>
+
+### SignatureSha256
+<details>
+ <summary>SignatureSha256 is recommended for appx or msix. It is the sha256 of signature file inside appx or msix. Could be used during streaming install if applicable</summary>
+
+ #### Optional Field
+
+ This key represents the signature file inside an MSIX installer. It is used to provide streaming install for MSIX packages.
```

Comment:
nit: we may specifically call out it's the sha256 hash of the AppxSignature.p7x file under package or bundle